packages/tizen_audio_manager/example/lib/volume_control.dart

```diff
@@ -0,0 +1,123 @@
+  int _currentVolume = 0;
+  int _maxVolume = 0;
+  Timer? _timer;
+  VolumeChangedEvent? _volumeChangedEvent;
+  StreamSubscription<VolumeChangedEvent>? _subscription;
+  final List<DropdownMenuItem<AudioVolumeType>> _dropdownButtonItems =
+      AudioVolumeType.values
+          .where((AudioVolumeType e) => e != AudioVolumeType.none)
+          .map((AudioVolumeType e) => DropdownMenuItem<AudioVolumeType>(
+              value: e, child: Text(e.toString().split('.').last)))
+          .toList();
+
+  @override
+  void initState() {
+    super.initState();
+    _onAudioTypeChanged(_selectedType);
+
+    _timer = Timer.periodic(const Duration(seconds: 1), (Timer timer) async {
+      final AudioVolumeType type =
+          await AudioManager.volumeController.currentPlaybackType;
+
+      if (type != _currentPlaybackType) {
+        setState(() {
+          _currentPlaybackType = type;
+        });
+      }
+    });
+
+    _subscription = AudioManager.volumeController.onChanged
+        .listen((VolumeChangedEvent event) {
+      setState(() {
+        _volumeChangedEvent = event;
+
+        if (event.type == _selectedType) {
+          _currentVolume = event.level;
+        }
+      });
+    });
+  }
+
+  @override
+  void dispose() {
+    _timer?.cancel();
+    _subscription?.cancel();
+    super.dispose();
+  }
+
+  Future<void> _onAudioTypeChanged(AudioVolumeType? type) async {
+    type = type ?? AudioVolumeType.ringtone;
+    final int currentVolume =
+        await AudioManager.volumeController.getLevel(type);
+    final int maxVolume = await AudioManager.volumeController.getMaxLevel(type);
+
+    setState(() {
+      _selectedType = type!;
+      _currentVolume = currentVolume;
+      _maxVolume = maxVolume;
+    });
+  }
+
+  void _onVolumeSliderChanged(double value) {
+    AudioManager.volumeController.setLevel(_selectedType, value.toInt());
+
+    setState(() {
+      _currentVolume = value.toInt();
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(title: const Text('Volume control')),
+      body: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: <Widget>[
+          Text('Current playback type: ' +
+              _currentPlaybackType.toString().split('.').last),
+          Row(mainAxisAlignment: MainAxisAlignment.center, children: <Widget>[
+            const Text('Audio type:'),
+            const SizedBox(width: 10),
+            DropdownButton<AudioVolumeType>(
+                value: _selectedType,
+                items: _dropdownButtonItems,
+                onChanged: _onAudioTypeChanged)
+          ]),
+          Text('Volume: $_currentVolume/$_maxVolume'),
+          SizedBox(
+              width: 250,
+              child: Slider(
+                value: _currentVolume.toDouble(),
+                min: 0,
+                max: _maxVolume.toDouble(),
+                onChanged: _onVolumeSliderChanged,
```

Comment:
You can use the `divisions` argument of [`Slider`](https://api.flutter.dev/flutter/material/Slider-class.html) to make the animation smoother.